esp_idf_panic_decoder/pc_address_decoder.py

```diff
@@ -11,6 +11,13 @@
 # regex matches an potential address
 ADDRESS_RE = re.compile(r'0x[0-9a-f]{8}', re.IGNORECASE)
 
+# regex matches address entries in addr2line output
```

Comment:
Could this comment tell why is another address RE created? Why has to be lookahead? Maybe make it also `re.IGNORECASE` or explain why this have to be case sensitive.